include/gc_inline.h

```diff
@@ -31,6 +31,10 @@
 #include "gc.h"
 #include "gc_tiny_fl.h"
 
+# ifdef _MSC_VER
+#   include <immintrin.h> /* for prefetch intrinsics */
```

Comment:
This is the wrong header. As far as I can tell, prefetch macros are defined in `intrin.h`.

Reply:
Thanks. Intel's intrinsic guide suggested this header, but on you are right that the MSVC docs point towards `intrin.h` -  see here: https://learn.microsoft.com/en-us/cpp/intrinsics/x64-amd64-intrinsics-list?view=msvc-170 -- will update!